### PR TITLE
SUM-114 - Adding video paragraph type

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY FOR REVIEW

# Summary
- Adding video paragraph
- https://stanfordits.atlassian.net/browse/SUM-114

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+uuid: f61f913e-0efd-42b5-947d-44baac63793d
```
Comment (pookmish) [inline docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml]: Can you delete these `next.next_entity_type_config.*` files. I gotta find out why the config ignore isn't working for them.

File: docroot/profiles/lagunita/summer_profile/config/summer/field.field.paragraph.sum_video.sum_youtube_url.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+uuid: 93f3b2dc-789f-46c5-9e66-5cea5c691727
```
Comment (pookmish) [inline docroot/profiles/lagunita/summer_profile/config/summer/field.field.paragraph.sum_video.sum_youtube_url.yml]: Make sure to expose the paragraph type and the field via graphql at `/admin/config/graphql_compose`

File: docroot/profiles/lagunita/summer_profile/config/summer/field.storage.paragraph.sum_youtube_url.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+uuid: 1229d88f-fece-44e4-a247-f8b1f2996f9d
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+    - paragraphs
+id: paragraph.sum_youtube_url
+field_name: sum_youtube_url
```
Comment (pookmish) [inline docroot/profiles/lagunita/summer_profile/config/summer/field.storage.paragraph.sum_youtube_url.yml]: make sure to namespace the fields with the bundle. in this case it should be `sum_video_video` or something. other fields will be like `sum_video_header` or such. See [dev guide documentation](https://sws-devguide.stanford.edu/standards-governance/naming-conventions)

**Review by pookmish — CHANGES_REQUESTED** [review]
Small changes needed.

File: docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml (hunk 1 above)
Comment (mdyoung3) [reply docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml]: Seem to have self-resolved with latest export.

File: docroot/profiles/lagunita/summer_profile/config/summer/field.field.paragraph.sum_video.sum_youtube_url.yml (hunk 2 above)
Comment (pookmish) [reply docroot/profiles/lagunita/summer_profile/config/summer/field.field.paragraph.sum_video.sum_youtube_url.yml:1]: Still incorrectly namespaced

File: docroot/profiles/lagunita/summer_profile/config/summer/field.storage.paragraph.sum_youtube_url.yml (hunk 3 above)
Comment (pookmish) [reply docroot/profiles/lagunita/summer_profile/config/summer/field.storage.paragraph.sum_youtube_url.yml]: Still incorrectly namespaced

File: docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml (hunk 1 above)
Comment (pookmish) [reply docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml]: They are still in the PR

**Review by pookmish — CHANGES_REQUESTED** [review]
Still needs the requested changes addressed

File: docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml (hunk 1 above)
Comment (mdyoung3) [reply docroot/profiles/lagunita/summer_profile/config/summer/next.next_entity_type_config.config_pages.stanford_basic_site_settings.yml]: Huh. Not sure what happened, but at one point they were gone. 🤷

File: docroot/profiles/lagunita/summer_profile/config/summer/field.storage.paragraph.sum_youtube_url.yml (hunk 3 above)
Comment (mdyoung3) [reply docroot/profiles/lagunita/summer_profile/config/summer/field.storage.paragraph.sum_youtube_url.yml]: This is really strange. I must have reverted or missed something. I know I fixed this.

Sorry.

**Comment by mdyoung3** [comment]
@pookmish Not sure what I did there. But I've double-checked and the changes are now in place for sure.  

Again, I apologize.

**Review by pookmish — APPROVED** [review]

**Pull request merged**